.github/workflows/create-pullrequest.yml

```diff
@@ -0,0 +1,19 @@
+name: Create or update pull request for non-master branches
```

Comment:
Can you explain what this action does? This is not quite clear to me...

Reply:
> Can you explain what this action does? This is not quite clear to me...

See here https://github.com/devops-infra/action-pull-request

Reply:
I must confess I don't understand this ... why is standard github Pullrequest not enough? This seems doing much more than "Adding a PR template" I certainly don't want that each and every branch is automatically created as a PR ... also all information added is already visible on the "Files changed" Tab already.